index.html

```diff
@@ -28,19 +28,192 @@ <h1 class="introduction">I'm Emmanuel, glad to see you</h1>
+        </section>
+        <section class="project-more">
+            <button class="works-see-more"><span class="more-text">See Project</span></button>
+        </section>  
+    </div>
+    
+    <div class="my-works">
+        <figure>
+            <img src="images/tonic-2-project.jpg" class="works-image" alt="Tonic App" width="100%"> 
+        </figure>
+        <div class="works-description">
+            <h3 class="my-works-project-title">Tonic</h3>
+            <ul class="works-headers">
+                <li class="title-header">CANOPY</li>
+                <li><img src="images/works-dot.PNG" class="dot-separator" alt="Dot separator"></li>
+                <li class="title-header-general">Back End Dev</li>
+                <li><img src="images/works-dot.PNG" class="dot-separator" alt="Dot separator"></li>
+                <li class="title-header-general">2015</li>
+            </ul>
+            <p class="project-description">
+                A daily selection of privately personalized reads; no accounts or sign-ups required.
+            </p>
+        </div>
+        <section class="project-languages">
+            <div class="coding-languages">html</div>
+            <div class="coding-languages">css</div>
+            <div class="coding-languages">javascript</div>
+        </section>
+        <section class="project-more">
+            <button class="works-see-more"><span class="more-text">See Project</span></button>
+        </section>  
+    </div>
+
+    <div class="my-works">
+        <figure>
+            <img src="images/multi-post-stories-2-project.jpg" class="works-image" alt="Tonic App" width="100%"> 
+        </figure>
+        <div class="works-description">
+            <h3 class="my-works-project-title">Multi-Post
+                Stories</h3>
+            <ul class="works-headers">
+                <li class="title-header">CANOPY</li>
+                <li><img src="images/works-dot.PNG" class="dot-separator" alt="Dot separator"></li>
+                <li class="title-header-general">Back End Dev</li>
+                <li><img src="images/works-dot.PNG" class="dot-separator" alt="Dot separator"></li>
+                <li class="title-header-general">2015</li>
+            </ul>
+            <p class="project-description">
+                A daily selection of privately personalized reads; no accounts or sign-ups required.
+            </p>
+        </div>
+        <section class="project-languages">
+            <div class="coding-languages">html</div>
+            <div class="coding-languages">css</div>
+            <div class="coding-languages">javascript</div>
+        </section>
+        <section class="project-more">
+            <button class="works-see-more"><span class="more-text">See Project</span></button>
+        </section>  
+    </div>
+    <!--My projects end//-->
+
+    <!-- About Myself Start-->
+    <section class="headline">
+        <h1 class="introduction">About Myself</h1>
 
```

Comment:
Kindly modify the styling of the `about myself` header to match your chosen design. You seem to have used a lesser font weight. Also notice how the two words are on different lines.

Your design:
![image](https://user-images.githubusercontent.com/20554583/192076302-78b413d2-1873-433f-a62a-d0b31976c203.png)

Figma Design:
![image](https://user-images.githubusercontent.com/20554583/192076309-97c08360-3ac5-4dbb-85cf-1e2a69e664a7.png)